editor/src/main/com/mbrlabs/mundus/ui/modules/Outline.java

```diff
@@ -129,8 +130,10 @@ public void onSceneChanged(SceneChangedEvent sceneChangedEvent) {
 
     @Override
     public void onSceneGraphChanged(SceneGraphChangedEvent sceneGraphChangedEvent) {
-        Log.traceTag(TAG, "SceneGraph changed. Building scene graph.");
-        buildTree(sceneGraph);
+//        Log.traceTag(TAG, "SceneGraph changed. Building scene graph.");
```

Comment:
if buildTree is fired on every change, the outline tree loose information on which node is expanded and which is closed. So when a new game object is added the outline tree is completely closed or expanded to maximum. This will confuse and frustrate user.

The other tools should have the same adding procedure to prevent this:
1. updating sceneGraph; 2. updating outline node

The other solution with rebuilding the tree every time is a not very clean solution.